articles/azure-app-configuration/faq.yml

```diff
@@ -146,15 +141,16 @@ sections:
+          All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature. When a standard tier App Configuration store is deleted, its name is reserved for the retention period. To recreate a store with the same name before the retention period expires, you need to [purge the soft-deleted store](./howto-recover-deleted-stores-in-azure-app-configuration#list-recover-or-purge-a-soft-deleted-app-configuration-store) first, provided the store doesn't have purge protection enabled.
 
-          Recreating an App Configuration store during the retention period is currently only supported by the Azure CLI using the `az appconfig create` [command](/cli/azure/appconfig#az_appconfig_create).
+          App Configuration previously offered an option to overwrite a soft-deleted store as instructed below. This option will be retired by April 30, 2022. Migrate to use the purge function or set a shorter retention period if you often need to recreate a store with the same name.
+          * It is possible to recreate a store with the same name before the retention period has elapsed, but it must be in the same subscription, resource group, and region as the original store. If the resource group that contained the store was deleted it will need to be recreated in the same subscription first before recreating the store inside of it.
+          * The App Configuration store cannot be created in a different subscription/resource group until the retention period has elapsed. However, the resource can be recreated in the original subscription/resource group and then moved to a new subscription/resource group by following [these instructions](../azure-resource-manager/management/move-resource-group-and-subscription.md#use-the-portal).
+          * Recreating an App Configuration store during the retention period is currently only supported by the Azure CLI using the `az appconfig create` [command](/cli/azure/appconfig#az_appconfig_create).
 
       - question: How can I restore an App Configuration store that I deleted mistakenly?
         answer: |
-          Standard tier App Configuration stores can be recovered within seven days of being deleted. To recover a mistakenly deleted App Configuration store contact Microsoft Support.
+          All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature. You can recover a soft-deleted store within its retention period. Follow these [instructions](./howto-recover-deleted-stores-in-azure-app-configuration) to recover a mistakenly deleted App Configuration store.
```

Comment:
"All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature." This reads more in the context of existing stores and means like we have enabled it now. As  we want to say this about all stores it should be "All App Configuration stores in the standard tier have this automatically enabled [soft-delete](./concept-soft-delete) feature."

Reply:
Do we have to specify "automatically enabled soft-delete feature", insinuates that we have a way to disable it?
With the below sentence do we miss anything in the meaning?

All App Configuration stores in the standard tier have this [soft-delete](https://github.com/MicrosoftDocs/azure-docs/pull/89696/concept-soft-delete) feature.

Reply:
I borrowed the sentence from the [concept doc](https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-soft-delete)  with a little rephrasing: "_It's automatically enabled for all stores in the standard tier._". If we do, we may have the same implication there.

Are you suggesting adding "this" as below? But the sentence doesn't read right, does it?
"All App Configuration stores in the standard tier have **this** automatically enabled [soft-delete](https://github.com/MicrosoftDocs/azure-docs/pull/concept-soft-delete) feature."

Reply:
I borrowed the sentence from the [concept doc](https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-soft-delete) with a little rephrasing: "It's automatically enabled for all stores in the standard tier.". If we do, we may have the same implication there.

Right about the same implications in concept doc, "automatically enabled" kind of throws user to - is there a disable? or how to disable? We should change that too :)

Reply:
I agree with what you said @Muksvso. How about we just make it explicit?

"All App Configuration stores in the standard tier have automatically enabled the [soft-delete](./concept-soft-delete) feature, which cannot be disabled."

Reply:
Or even simpler:

"All App Configuration stores in the standard tier support the [soft-delete](./concept-soft-delete) feature, which cannot be disabled."

Reply:
Updated the PR with a more explicit statement. I also changed from "soft-deleted store" to "deleted store" as you suggested.